### ISSUE TITLE
Scroll all tabs not working

### ISSUE THREAD

**Issue by Heshyo** [onemen/TabMixPlus#333]
Using the mouse wheel to scroll between the lines of tabs does not work. I have to drag the scroll bar that appears.

Mouse wheel while holding Shift does properly change the selected tab.

I tried changing the default action when scrolling over the tab-bar, but I still had the same issue.

FF 131.0 (64-bit)
Release v1.23.0

![image](https://github.com/user-attachments/assets/892abbbb-4b43-4ecf-8182-9e28433da04d)

**Comment by onemen** [comment]
Attach a file with your Tab mix plus preference

**Comment by Heshyo** [comment]
The scroll bar is the blue thing on the right of the tabs. If I drag it, then I can go up or down the tabs.

![image](https://github.com/user-attachments/assets/03d326c0-866a-4731-87dc-8dae259e4851)

[TMPpref.txt](https://github.com/user-attachments/files/17253837/TMPpref.txt)

**Comment by onemen** [comment]
This scroll bar is not from Tab Mix Plus, you probably installed another extension or script to handle multi-row.

If you provide a link to that extension or script i will check for ways to improve compatibility

Read in our [docs](https://onemen.github.io/tabmixplus-docs/help/display-tab-bar/) how to set Tab Mix Plus multi-row

**Comment by Heshyo** [comment]
You're perfectly right. Years ago I stopped using TabMixPlus and had to switch to https://github.com/Izheil/Quantum-Nox-Firefox-Dark-Full-Theme But it stopped working after the latest FF update, and I just noticed TMP was still alive and working. So I installed it, but didn't remove all modifications from the other plugin. I've now removed them and set TMP to show 3 rows of tab. Everything is working again.
Thanks a lot for your reactivity and your work over the years!
